src/meshes/triangle_mesh.template.cc

```diff
@@ -30260,6 +30260,10 @@ namespace oomph
       // Set the element area
       triangle_mesh_parameters.element_area() = max_area;
 
+      // Set the minimum element interior angle.
+      triangle_mesh_parameters.set_min_element_interior_angle(
+        this->Min_element_interior_angle_for_adaption);
```

Comment:
Min_element_interior_angle_after_adaptation is (I think) more accurate.

Reply:
The angle is required for the adaption step. Perhaps Min_element_interior_angle_during_adaption is better, or to just drop the adaption part: Min_element_interior_angle.

Reply:
Min_permitted_element_interior_angle would be better still as it is a tolerance rather than a measurement.